lib/server.js

```diff
@@ -218,6 +218,11 @@ function Server (torrent, opts = {}) {
       const html = getPageHTML('405 - Method Not Allowed', '<h1>405 - Method Not Allowed</h1>')
       res.end(html)
     }
+
+    function serveEmptyResponse () {
+      res.statusCode = 204
```

Comment:
Ideally, we'd just kill the whole TCP connection without responding at all but I'm not sure if there's a clean way to do this from the http layer. Looking into it.